client/tx/search.go

```diff
@@ -27,7 +27,20 @@ const (
 func SearchTxCmd(cdc *wire.Codec) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "txs",
-		Short: "Search for all transactions that match the given tags",
+		Short: "Search for all transactions that match the given tags.",
+		Long: strings.TrimSpace(`
+Search for transactions that match the given tags. By default, transactions must match ALL tags 
+passed to the --tags option. To match any transaction, use the --any option.
+
+For example:
+
+$ gaiacli advanced tendermint txs --tag test1,test2
```

Comment:
note `advanced` will soon be removed https://github.com/cosmos/cosmos-sdk/pull/2014